server/controller/src/main/java/ai/starwhale/mlops/domain/project/mapper/ProjectMapper.java

```diff
@@ -59,7 +60,7 @@ public interface ProjectMapper {
             + " where id = #{id}")
     ProjectEntity find(@Param("id") Long id);
 
-    @Select("select " + COLUMNS + " from project_info"
+    @Select("select " + SIMPLIFY_COLUMNS + " from project_info"
             + " where project_name = #{projectName}"
             + " and is_deleted = 0")
     List<ProjectEntity> findByName(@Param("projectName") String projectName);
```

Comment:
We will get different results when using the `/api/v1/project/{projectUrl}` API if the projectUrl is id or name.

https://github.com/star-whale/starwhale/blob/af283add71da71cae6a174af566075192c0afe4c/server/controller/src/main/java/ai/starwhale/mlops/domain/project/ProjectDao.java#L57-L77

Reply:
how about adding a devoting API to fetch overview, since it's a longtext?

Reply:
done
![image](https://github.com/star-whale/starwhale/assets/24869618/c71edecb-f688-4fc9-baef-360d69a01df9)